### ISSUE TITLE
The speed of detect is more lower than the tutorial

### ISSUE THREAD

**Issue by klyjm** [ultralytics/yolov5#1192]
## ❔Question
I have used the model to detect on TITAN V which is faster than TESLA P100, but I found the speed (0.018s) is slower 2x than the speed in the tutorial(0.009s). Is the speed of yolov5 really so fast?

## Additional context
```Shell
python detect.py --source ./inference/images/ --weights yolov5s.pt --conf 0.4 --device 3
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.4, device='3', img_size=640, iou_thres=0.5, output='inference/output', save_txt=False, source='./inference/images/', update=False, view_img=False, weights=['yolov5s.pt'])
Using CUDA device0 _CudaDeviceProperties(name='TITAN V', total_memory=12066MB)

Fusing layers...
Model Summary: 140 layers, 7.45958e+06 parameters, 0 gradients
image 1/2 inference/images/bus.jpg: 640x512 4 persons, 1 buss, Done. (0.018s)
image 2/2 inference/images/zidane.jpg: 384x640 2 persons, 1 ties, Done. (0.017s)
Results saved to inference/output
Done. (0.173s)

**Comment by glenn-jocher** [comment]
@klyjm speeds and exact commands to reproduce are clearly shown in the README:
https://github.com/ultralytics/yolov5#pretrained-checkpoints

**Comment by klyjm** [comment]
> @klyjm speeds and exact commands to reproduce are clearly shown in the README:
> https://github.com/ultralytics/yolov5#pretrained-checkpoints

No, what I mean is in the Tutorial, run the detect.py is 0.009ms per image:
```Shell
$ python detect.py --source ./inference/images/ --weights yolov5s.pt --conf 0.4

Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.4, device='', fourcc='mp4v', half=False, img_size=640, iou_thres=0.5, output='inference/output', save_txt=False, source='./inference/images/', view_img=False, weights='yolov5s.pt')
Using CUDA device0 _CudaDeviceProperties(name='Tesla P100-PCIE-16GB', total_memory=16280MB)

Downloading https://drive.google.com/uc?export=download&id=1R5T6rIyy3lLwgFXNms8whc-387H0tMQO as yolov5s.pt... Done (2.6s)

image 1/2 inference/images/bus.jpg: 640x512 3 persons, 1 buss, Done. (0.009s)
image 2/2 inference/images/zidane.jpg: 384x640 2 persons, 2 ties, Done. (0.009s)
Results saved to /content/yolov5/inference/output
```
But I run the detect.py with same config on TITAN V can't reach the speed, just like showing before. I know we can acclerate the speed of test.py by some ways, just like using bigger batch-size. But it looks like the speed shown by detect.py is slower in actual.

**Comment by glenn-jocher** [comment]
@klyjm tutorial screenshots are taken directly from GCP or Colab instances. If you fail to reproduce results in README please advise, otherwise comparing speeds across different hardware is not very informative.

**Comment by klyjm** [comment]
> @klyjm tutorial screenshots are taken directly from GCP or Colab instances. If you fail to reproduce results in README please advise, otherwise comparing speeds across different hardware is not very informative.

Yeah, I know. But it is slower in a better GPU ( 0.018s on ours TITIAN V vs 0.009s on TESLA P100 in screenshot), it's so strange. I am just intresting in why inference in detect.py is slower in a better GPU. So this is just a question.

**Comment by glenn-jocher** [comment]
@klyjm our speed tests measure average FPS over the 5000 images in the COCO test set with commands to reproduce here:
https://github.com/ultralytics/yolov5#pretrained-checkpoints

If you can not reproduce with the same GPU please advise, otherwise there is no action for us to take.

**Comment by glenn-jocher** [comment]
** SpeedGPU measures end-to-end time per image averaged over 5000 COCO val2017 images using a GCP n1-standard-16 instance with one V100 GPU, and includes image preprocessing, PyTorch FP16 image inference at --batch-size 32 --img-size 640, postprocessing and NMS. Average NMS time included in this chart is 1-2ms/img. Reproduce by python test.py --data coco.yaml --img 640 --conf 0.1

**Comment by leesdust** [comment]
@klyjm Do you already know why?

**Comment by klyjm** [comment]
> @klyjm Do you already know why?

No. I can't acclerate speed of detect.py on my better GPU to the same speed in the toturial. Maybe the model of the toturial is different with today's model.
